source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/product-definition-select/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { compose } from 'recompose';
+
+import { withTranslations } from '@lib/i18n';
+import { withNetwork as withProductDefinitions } from '@data/containers/resources/product-definition/list';
+import { withLoader } from '@data/containers/with-loader';
+
+import { Display } from './display';
+
+export { Display } from './display';
```

Comment:
any particular reason why to export the display?

Reply:
like with the org-select, and project-table, if someone wants to build out their own hook ups to it